cli/pkg/kctrl/cmd/app/release/release.go

```diff
@@ -22,11 +22,11 @@ type ReleaseOptions struct {
 	chdir          string
 	outputLocation string
 	debug          bool
+	tag            string
 }
 
 const (
 	defaultArtifactDir = "carvel-artifacts"
-	defaultVersion     = "0.0.0+build.%d"
```

Comment:
Why are we removing this?
This is supposed to be the valid server for a build and should be the value of `spec.Version` when no version is specified.

Reply:
We can see this in line 70 are we moving this elsewhere?

Reply:
Oh nvm, I mistook this for the package/release package
Looks like we do not need this here